src/org/lsst/ts/jenkins/components/Csc.groovy

```diff
@@ -332,23 +332,28 @@ def update_container_branches() {
     withEnv([]){
         sh """
         set +x
+	shopt -s extglob
         source /home/saluser/.setup_dev.sh || echo loading env failed. Continuing...
 
+	# Update branches internal to container
         for repo in \$(ls /home/saluser/repos/)
         do
             cd /home/saluser/repos/\$repo
             /home/saluser/.checkout_repo.sh ${WORK_BRANCHES}
-            git pull
         done
+	cd ${env.WORKSPACE}/ci/
+	# Deal with some file oddities
```

Comment:
Please explain that you are deleting extraneous files that you don't want to loop over in the next step.